programming-language-resources.md

```diff
@@ -59,12 +59,15 @@
 - HTML
     - [Quick HTML Tutorial for Beginners](https://www.youtube.com/playlist?list=PLr6-GrHUlVf_ZNmuQSXdS197Oyr1L9sPB)
     - [HTML Crash Course for Beginners (1 hr)](https://www.youtube.com/watch?v=UB1O30fR-EE)
+    - [Basic HTML and HTML5](https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/)
 - CSS
     - [Quick CSS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLr6-GrHUlVf8JIgLcu3sHigvQjTw_aC9C)
     - [CSS Crash Course for absolute Beginners](https://www.youtube.com/watch?v=yfoY53QXEnI)
+    - [Basic CSS](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/)
 - Javascript
     - [Learn Javascript in 12 minutes](https://www.youtube.com/watch?v=Ukg_U3CnJWI)
     - [Javascript Beginner Tutorials](https://www.youtube.com/playlist?list=PL41lfR-6DnOrwYi5d824q9-Y6z3JdSgQa)
+    - [Javascript Algorithms and Data Structures](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/)
 - Other Language 1
```

Comment:
[Python Full Course - Learn Python in 12 Hours | Python Tutorial For Beginners | Edureka](https://youtu.be/WGJJIrtnfpk)

Reply:
This is a pull request for adding links to html css and javascript course. But you have mentioned a python course. Maybe you can create a new branch and pull request for adding this link